ui/components/MesheryPlayComponent.js

```diff
@@ -75,98 +75,87 @@ const styles = (theme) => ({
   },
 });
 
-class MesheryPlayComponent extends React.Component {
-  constructor(props) {
-    super(props);
+function MesheryPlayComponent(props) {
```

Comment:
- Can we use Arrow function here?
```suggestion
const MesheryPlayComponent = (props) => {
```

Reply:
The rest looks good to me 🚀